public/javascripts/data/paragraph.js

```diff
@@ -0,0 +1,32 @@
+const Database = require('better-sqlite3');
+
+
+// Function writes paragraph to the database
+exports.insertParagraph = (paragraph) => {
+
+    // TODO:
+    // 1. (user_id, stories_id) should contain current user and current story
+    const db = new Database('controllers/stories.db');
+    const new_paragraph = db.prepare('INSERT INTO paragraphs (user_id, stories_id, content) VALUES (?, ?, ?)');
+    const values = new_paragraph.run(4, 1, paragraph);
+    db.close();
+    return;
+};
+
+// Function gets all paragraphs in the database, stores them in array
+exports.displayAllParagraphs = () => {
+    
+    // TODO:
+    // 1. (stories_id) should contain the current story_id where the POST request was sent from
+    const db = new Database('controllers/stories.db');
```

Comment:
Consider moving your database declaration to your `require` on line 1:

`const db = require('better-sqlite3')('contollers/stories.db');`

I'd also move `stories.db` outside of controllers, and probably into the project root or the `/data` folder